src/MainWindow.vala

```diff
@@ -45,81 +40,77 @@ namespace Quilter {
         public const string ACTION_EXPORT_PDF = "action_export_pdf";
         public const string ACTION_EXPORT_HTML = "action_export_html";
         public static Gee.MultiMap<string, string> action_accelerators = new Gee.HashMultiMap<string, string> ();
-
+        
         private const GLib.ActionEntry[] action_entries = {
             { ACTION_CHEATSHEET, action_cheatsheet },
             { ACTION_PREFS, action_preferences },
             { ACTION_EXPORT_PDF, action_export_pdf },
             { ACTION_EXPORT_HTML, action_export_html }
         };
-
+        
         public bool is_fullscreen {
             get {
-                var settings = AppSettings.get_default ();
-                return settings.fullscreen;
+                return gsettings.get_boolean("fullscreen");
             }
             set {
-                var settings = AppSettings.get_default ();
-                settings.fullscreen = value;
-
-                if (settings.fullscreen) {
+                gsettings.set_boolean("fullscreen", value);
+                if (value) {
                     fullscreen ();
-                    settings.statusbar = false;
-		            settings.sidebar = false;
+                    gsettings.set_boolean("statusbar", false);
+                    gsettings.set_boolean("sidebar", true);
```

Comment:
It looks like you changed the logic here. Intentional?

Reply:
Intentional, some UI to let the user change documents even when fullscreen.